src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
```

Comment:
Let's add such a constant to avoid magic values.

```suggestion
const MALE = 'm';
const FEMALE = 'f';

export const Person = ({ person }) => (
```